docs/core/tutorials/cli-templates-create-item-template.md

```diff
@@ -110,6 +110,7 @@ Open the _template.json_ with your favorite text editor and paste in the followi
   "identity": "ExampleTemplate.StringExtensions",
   "name": "Example templates: string extensions",
   "shortName": "stringext",
+  "sourceName":"ExampleTemplate.StringExtensions",
```

Comment:
This change doesn't make sense here. The item template doesn't have a source name to actually change.